Marker/Marker/Utility/Extensions/LineBreakMode/LineBreakMode.swift

```diff
@@ -11,5 +11,5 @@
     public typealias LineBreakMode = NSLineBreakMode
 #elseif os(macOS)
     import AppKit
-    public typealias LineBreakMode = NSParagraphStyle.LineBreakMode
+    public typealias LineBreakMode = NSLineBreakMode
```

Comment:
it might be worthwhile to get rid of this shimming, and replace with just line 14 since they now behave the same on the 3 OSes